.github/workflows/deploy-labels.yml

```diff
@@ -9,6 +9,10 @@ on:
       - .github/workflows/deploy-labels.yml
   workflow_dispatch:
 
+permissions:
+  contents: read
+  issues: write
```

Comment:
@bengeois or @Tarektouati  do you know why "issues" ? Labal management is under "issues" rights ?